setup/setup.sh

```diff
@@ -126,14 +134,11 @@ installDocker() {
 installDocker
```

Comment:
Where gcloud is installed? This is prerequisites now just like Docker.

Reply:
Sure, I'll add it to README.